deeppavlov/configs/ranking/paraphrase_ident_paraphraser_interact.json

```diff
@@ -0,0 +1,124 @@
+{
+  "dataset_reader": {
+    "name": "paraphraser_reader",
+    "num_samples": null,
+    "seed": 243,
+    "data_path": "./paraphraser_data"
+  },
+  "dataset_iterator": {
+    "name": "siamese_iterator",
+    "seed": 243
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y"],
+    "pipe": [
+      {
+        "id": "preproc",
+        "name": "siamese_preprocessor",
+        "use_matrix": false,
+        "max_sequence_length": 28,
+        "fit_on": ["x"],
+        "in": ["x"],
+        "out": ["x_proc"],
+        "sent_vocab":
+        {
+          "id": "siam_sent_vocab",
+          "name":"simple_vocab",
+          "save_path": "paraphraser_vocabs/sent.dict",
+          "load_path": "paraphraser_vocabs/sent.dict"
+        },
+        "tokenizer":
+        {
+          "name": "nltk_tokenizer"
+        },
+        "vocab":
+        {
+          "id": "siam_vocab",
+          "name":"simple_vocab",
+          "save_path": "paraphraser_vocabs/tok.dict",
+          "load_path": "paraphraser_vocabs/ctok.dict"
```

Comment:
Is it intended that save and load paths here are different?

Reply:
Other configs are affected too

Reply:
corrected, thank you!